isis/tests/BundleMeasureTests.cpp

```diff
@@ -0,0 +1,166 @@
+#include "BundleControlPoint.h"
+#include "BundleMeasure.h"
+#include "BundleSettings.h"
+#include "ControlMeasure.h"
+#include "ControlPoint.h"
+#include "IException.h"
+#include "SpecialPixel.h"
+
+#include "gmock/gmock.h"
+
+// To be used for basic setup of a BundleMeasure test object. (Fixture Function)
+class BundleMeasure_CreateObj : public ::testing::Test
+{
+
+  protected:
+    // Create a ControlPoint object using the default constructor
+    // to feed its pointer into the BundleControlPoint constructor.
+    ControlPoint controlPoint;
+
+    // Create a ControlMeasure object using the default constructor
+    // to feed its pointer into the BundleMeasure constructor.
+    ControlMeasure controlMeasure;
+
+    // Since BundleMeasure and BundleControlPoint do not have
+    // default constructors, and C++ does not allow for object declarations
+    // pointers to each respective object are used.
+    BundleMeasure *testBundleMeasurePtr;
+    BundleControlPoint *testBundleControlPointPtr;
+
+
+    void SetUp() override
+    {
+      // BundleControlPoint testBundleControlPoint;
+      // Create a BundleSettingsQsp to feed it to
+      // the BundleMeasure constructor.
+      BundleSettingsQsp bundleSettingsQsp( new BundleSettings );
+
+      // Create a BundleControlPoint object to feed its pointer into
+      // the BundleMeasure constructor.
+      BundleControlPoint bundleControlPoint( bundleSettingsQsp, &controlPoint );
+
+      // Finally, the SetUp function creates the BundleMeasure object
+      // from all of the objects made previously.
+      // std::cout << &controlMeasure << '\n';
+      BundleMeasure bundleMeasure( &controlMeasure, &bundleControlPoint );
```

Comment:
Both of these objects will be destroyed when they go out of scope once the setup function returns. I'd expect this to cause your first test to seg fault.

You should heap allocate them instead of stack allocate them. I'd also recommend using a smart pointer such as a unique_ptr.